### PR TITLE
Add Web Share to standalone feature page

### LINKED ISSUE #505: Add webshare API for sharing each feature
https://www.chromestatus.com/feature/5668769141620736

Why not!?

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Fix #505

### CODE REVIEW

File: static/sass/shared.scss [hunk 1]
```diff
@@ -77,3 +77,7 @@ button:not(:disabled):active {
     display: initial;
   }
 }
+
+.no-web-share {
+  display: none;
```
Comment (ebidel) [inline static/sass/shared.scss]: want to combine this with the `.no-push-notifications` selector?

File: static/sass/shared.scss [hunk 2]
```diff
@@ -77,3 +77,7 @@ button:not(:disabled):active {
     display: initial;
   }
 }
+
+.no-web-share {
```
Comment (ebidel) [inline static/sass/shared.scss]: want to combine this with the `.no-push-notifications` selector?

File: templates/feature.html [hunk 3]
```diff
@@ -52,6 +52,12 @@
         <iron-icon icon="chromestatus:bug-report"></iron-icon>
       </a>
     </span>
+    <span class="tooltip no-web-share" title="Share this feature">
+      <a href="#" data-tooltip>
+        <iron-icon icon="chromestatus:share"
```
Comment (ebidel) [inline templates/feature.html]: single line is fine here.

File: templates/feature.html [hunk 4]
```diff
@@ -241,9 +247,34 @@
   }
 }
 
+function shareFeature() {
+  if (navigator.share) {
+    var url = '/feature/{{ feature.id }}';
+    navigator.share({
+        title: '{{ feature.name }}',
```
Comment (ebidel) [inline templates/feature.html]: 2 space indent

**Comment by ebidel** [comment]
What do you think about adding this to https://www.chromestatus.com/features/schedule and /features?

File: static/sass/shared.scss (hunk 2 above)
Comment (beaufortfrancois) [reply static/sass/shared.scss]: Done.

File: templates/feature.html (hunk 3 above)
Comment (beaufortfrancois) [reply templates/feature.html]: Done.

File: templates/feature.html (hunk 4 above)
Comment (beaufortfrancois) [reply templates/feature.html]: Done.

**Comment by beaufortfrancois** [comment]
Feedback addressed.

I'm afraid it will clutter the UI with too many icons if we include it to `/features` and `/features/schedule`

**Comment by ebidel** [comment]
Ack. We can see how this gets used on /feature pages first. Thanks for crushing this out so quickly!

**Pull request merged**